flow/metrics.yaml

```diff
@@ -69,3 +81,44 @@ components:
               x-field-uid: 1
             cut_through:
               x-field-uid: 2
+    Flow.TxRxReplication:
+      description: |-
+        Tx Rx replication factor is required to determine the ratio of expected number of Rx packets across all Rx ports
+        to the Tx packets for the configured flow.
+      type: object
+      properties:
+        choice:
+          type: string
+          default: one
+          x-field-uid: 1
+          x-enum:
+            one:
+              x-field-uid: 1
+            rx_ports_count:
+              x-field-uid: 2
+            custom:
+              x-field-uid: 3
+        one:
+          $ref: '#/components/schemas/Flow.TxRxReplication.Ecmp'
+          x-field-uid: 2
+        rx_ports_count:
+          $ref: '#/components/schemas/Flow.TxRxReplication.RxCount'
+          x-field-uid: 3
+        custom:
+          description: |-
```

Comment:
I'm not able to fully follow this description and what is the use-case for this.  Also refrain from "we may need to specify", instead state it as "A custom integer value (>1) can be specified for loss calculations in scenarios where there are multiple ..."

Reply:
I have updated the description based on suggestion. Regarding how the Custom enum is used refer to the mail with Alex's example.